docs/user/management.asciidoc

```diff
@@ -7,117 +7,166 @@
+flushing, and clearing the cache. Practicing good index management ensures
+that your data is stored cost effectively.
 
-Manage your remote clusters for use with cross-cluster search and cross-cluster replication.
-You can add and remove remote clusters, and check their connectivity.
+| <<index-lifecycle-policies, Index Lifecycle Policies>>
+|Create a policy for defining the lifecycle of an index as it ages
+through the hot, warm, cold, and delete phases.
+Such policies help you control operation costs
+because you can put data in different resource tiers.
 
-| <<data-rollups, *Rollup Jobs*>>
+| <<snapshot-repositories, Snapshot and Restore>>
+|Define a policy that creates, schedules, and automatically deletes snapshots to ensure that you
+have backups of your cluster in case something goes wrong.
 
-Create a job that periodically aggregates data from one or more indices, and then
+| <<data-rollups, Rollup Jobs>>
+|Create a job that periodically aggregates data from one or more indices, and then
 rolls it into a new, compact index. Rollup indices are a good way to store months or
 years of historical data in combination with your raw data.
 
-| <<snapshot-repositories, *Snapshot and Restore*>>
+| {ref}/transforms.html[Transforms]
+|Use transforms to pivot existing {es} indices into summarized or entity-centric indices.
 
-Define a policy that creates, schedules, and automatically deletes snapshots to ensure that you
-have backups of your cluster in case something goes wrong.
+| <<managing-cross-cluster-replication, Cross-Cluster Replication>>
+|Replicate indices on a remote cluster and copy them to a follower index on a local cluster.
+This is important for
+disaster recovery. It also keeps data local for faster queries.
 
-| {ref}/transforms.html[*Transforms*]
+| <<working-remote-clusters, Remote Clusters>>
+|Manage your remote clusters for use with cross-cluster search and cross-cluster replication.
+You can add and remove remote clusters, and check their connectivity.
+|===
 
-Use transforms to pivot existing {es} indices into summarized or entity-centric indices.
+[float]
+[[manage-alerts-insights]]
+== Alerts and Insights
 
-| <<upgrade-assistant, *Upgrade Assistant*>>
+[cols="50, 50"]
+|===
 
-Identify the issues that you need to address before upgrading to the
-next major version of {es}, and then reindex, if needed.
+| <<managing-alerts-and-actions, Alerts&nbsp;and Actions>>
+| Centrally manage your alerts across {kib}. Create and manage reusable
+connectors for triggering actions.
 
-| <<watcher-ui, *Watcher*>>
+| <<reporting-getting-started, Reporting>>
+| Monitor the generation of reports&mdash;PDF, PNG, and CSV&mdash;and download reports that you previously generated.
+A report can contain a dashboard, visualization, saved search, or Canvas workpad.
+
+| <<xpack-ml-anomalies, Machine Learning Jobs>>
+| Create jobs that model the normal behavior of your time series data &mdash;
```

Comment:
It looks like you can only view the list of jobs and launch the ML UIs from here, so I suggest something like this:
```suggestion
| View your {anomaly-jobs} and {dfanalytics-jobs}. Open the Single Metric Viewer or Anomaly Explorer to see your {ml} results.
```